lib/optimizer/src/Transformer/ServerSideRendering.php

```diff
@@ -64,41 +140,40 @@ public function transform(Document $document, ErrorCollection $errors)
          * we also look for reasons not to remove the boilerplate.
          */
         $canRemoveBoilerplate = true;
-        foreach ($document->ampElements as $amp_element) {
+        foreach ($document->ampElements as $ampElement) {
             // Make sure we only deal with valid elements.
-            if (!$amp_element instanceof DOMElement) {
+            if (! $ampElement instanceof DOMElement) {
                 continue;
             }
 
             // Skip tags inside a template tag.
-            if ($this->hasAncestorWithTag($amp_element, Tag::TEMPLATE)) {
+            if ($this->hasAncestorWithTag($ampElement, Tag::TEMPLATE)) {
                 continue;
             }
 
             /*
-             * If these attributes are used on any AMP custom element tags within the document, we can't remove the
-             * boilerplate - they require the boilerplate.
+             * amp-experiment is a render delaying extension iff the tag is used in the doc. We check for that here
```

Comment:
```suggestion
             * amp-experiment is a render delaying extension if the tag is used in the doc. We check for that here
```

Reply:
`iff` is commonly used for "if and only if" and is also what is being used in the Node.js code from where this is taken from.

Reply:
Makes sense. I wasn't sure if that was the case, or if it was a typo.